stepguider_sxao_indi.cpp

```diff
@@ -371,58 +398,65 @@ void StepGuiderSxAoINDI::ShowPropertyDialog()
     SetupDialog();
 }
 
-void StepGuiderSxAoINDI::IndiServerConnected()
+void StepGuiderSxAoINDI::serverConnected()
```

Comment:
what would be the new 2.0 way to check if server connected is really true ?

Reply:
Many code fragments were not really required. This fragment was adding unnecessary delay when it should have been instant. Now it is *very* fast to connect to INDI unlike before.